### ISSUE TITLE
Endplate sclerosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4871]
**HPO term**
https://hpo.jax.org/app/browse/term/HP:0004576

**Synonym to be added**
Endplate sclerosis (may actually be the better primary label)

**broad/exact/narrow/plural/abbreviation ?**
exact
